README.adoc

```diff
@@ -1750,6 +1750,12 @@ Tools that pair nicely with `just` include:
 
 For lightning-fast command running, put `alias j=just` in your shell's configuration file.
 
+The aliased command may not keep the shell completion functionality described in the next section. In Bash, you can use the same completion function as `just` for your aliased command.
```

Comment:
This works in `zsh`, so I'm thinking this might be a bash specific issue, so it would be useful to call out Bash early on.

```suggestion
In `bash`, the aliased command may not keep the shell completion functionality described in the next section. Add the following line to your `.bashrc` to use the same completion function as `just` for your aliased command:
```